### PR TITLE
Card block styling

### LINKED ISSUE #10: Cards Block
Implement styling for the existing cards block

Screenshot 2023-08-02 at 16 59 36

### PR DESCRIPTION
**Pull request by deepakk799**
Please always provide the [GitHub issue(s)](../issues/10) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #10 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://10-cards-block--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/cards/cards.js [hunk 1]
```diff
@@ -3,14 +3,30 @@ import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
 export default function decorate(block) {
   /* change to ul, li */
   const ul = document.createElement('ul');
+
   [...block.children].forEach((row) => {
     const li = document.createElement('li');
+    const heading = row.querySelector('h2');
+    heading.className = 'card-title';
+    heading.remove();
```
Comment (mhaack) [inline blocks/cards/cards.js:11]: If you remove the `heading` there is no need to set a className

File: tools/actions/package-lock.json
Comment (mhaack) [inline tools/actions/package-lock.json:1]: Why are there changes here, please revert.

File: blocks/cards/cards.js [hunk 2]
```diff
@@ -3,14 +3,30 @@ import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
...
+    const link = row.querySelector('a');
+    link.className = 'card-link';
+    link.innerHTML += ' &rarr;';
+    link.remove();
```
Comment (mhaack) [inline blocks/cards/cards.js:15]: Same as above, if you remove the `link` there is no need to set a className or innerHTML

**Comment by duynguyen** [comment]
@deepakk799 just wondering why you write new css instead of using tailwind?

File: blocks/cards/cards.js (hunk 1 above)
Comment (deepakk799) [reply blocks/cards/cards.js:11]: Thanks @mhaack  for your feedback, I am using the heading element to append at line# 26. I had to do that as we wanted that to be outside of the li element. Let me know if there is any better way to do that.

File: blocks/cards/cards.js (hunk 2 above)
Comment (deepakk799) [reply blocks/cards/cards.js:15]: I'm using the link element as well outside of the original tag at line# 29.

**Comment by deepakk799** [comment]
@duynguyen  I'm using tailwind classes, not writing my own CSS. if you look at css file I'm using @apply directive for the classes. Let me know if that's not correct. Thank you.

e.g. 

.cards .card-title{
  @apply text-gray-900 my-2 font-extrabold text-3xl py-2;
}

File: blocks/cards/cards.js (hunk 1 above)
Comment (mhaack) [reply blocks/cards/cards.js]: This looks good now

**Review by mhaack — APPROVED** [review]
LGTM

**Pull request merged**